src/resources/views/crud/filters/date_range.blade.php

```diff
@@ -101,20 +103,54 @@ function applyDateRangeFilter{{camel_case($filter->name)}}(start, end) {
-		            'Last Month': [moment().subtract(1, 'month').startOf('month'), moment().subtract(1, 'month').endOf('month')]
+		            '{{trans('backpack::crud.today')}}': [moment().startOf('day'), moment().endOf('day')],
+		            '{{trans('backpack::crud.yesterday')}}': [moment().subtract(1, 'days'), moment().subtract(1, 'days')],
+		            '{{trans('backpack::crud.last_7_days')}}': [moment().subtract(6, 'days'), moment()],
+		            '{{trans('backpack::crud.last_30_days')}}': [moment().subtract(29, 'days'), moment()],
+		            '{{trans('backpack::crud.this_month')}}': [moment().startOf('month'), moment().endOf('month')],
+		            '{{trans('backpack::crud.last_month')}}': [moment().subtract(1, 'month').startOf('month'), moment().subtract(1, 'month').endOf('month')]
 		        },
 				@if ($filter->currentValue)
 		        startDate: moment("{{ $start_date }}"),
 		        endDate: moment("{{ $end_date }}"),
-				@endif
+                @endif
+                locale: {
+                    "format": "{{ $dateFormat }}",
+                    "applyLabel": "{{trans('backpack::crud.apply')}}",
+                    "cancelLabel": "{{trans('backpack::crud.cancel')}}",
+                    "fromLabel": "{{trans('backpack::crud.from')}}",
+                    "toLabel": "{{trans('backpack::crud.to')}}",
+                    "customRangeLabel": "{{trans('backpack::crud.custom_range')}}",
+                    "weekLabel": "{{trans('backpack::crud.week_label')}}",
+                    "daysOfWeek": [
+                        "{{trans('backpack::crud.short_sunday')}}",
+                        "{{trans('backpack::crud.short_monday')}}",
+                        "{{trans('backpack::crud.short_tuesday')}}",
+                        "{{trans('backpack::crud.short_wednesday')}}",
+                        "{{trans('backpack::crud.short_thursday')}}",
+                        "{{trans('backpack::crud.short_friday')}}",
+                        "{{trans('backpack::crud.short_saturday')}}"
+                    ],
+                    "monthNames": [
+                        "{{trans('backpack::crud.january')}}",
+                        "{{trans('backpack::crud.february')}}",
+                        "{{trans('backpack::crud.march')}}",
+                        "{{trans('backpack::crud.april')}}",
+                        "{{trans('backpack::crud.may')}}",
+                        "{{trans('backpack::crud.june')}}",
+                        "{{trans('backpack::crud.july')}}",
+                        "{{trans('backpack::crud.august')}}",
+                        "{{trans('backpack::crud.september')}}",
+                        "{{trans('backpack::crud.october')}}",
+                        "{{trans('backpack::crud.november')}}",
+                        "{{trans('backpack::crud.december')}}"
+                    ],
+                    "firstDay": {{var_export($filterOptions['weekFirstDay'] ?? 0)}}
```

Comment:
@pxpm since the JS label here is ```firstDay```, not ```weekFirstDay``` - wouldn't it be best to keep it ```firstDay``` in PHP too? So that people could define the PHP array as similar as possible to the JS array?

Reply:
OMG! Sometimes my brain just stops. Sure it makes all sense to have both keys named accordingly. I will push a fix to this :+1: